### ISSUE TITLE
[v1] parse error: Invalid numeric literal at line 2, column 0

### ISSUE THREAD

**Issue by arkivanov** [nexus-actions/create-nexus-staging-repo#11]
The action `v1` started to fail with the following error:

```
Warning: Unexpected input(s) 'staging-profile-id', valid inputs are ['entryPoint', 'args', 'username', 'password', 'staging_profile_id', 'description', 'base_url']
Run nexus-actions/create-nexus-staging-repo@v1
  with:
    username: arkivanov
    password: ***
    staging-profile-id: ***
    description: Created by GitHub Actions
    base_url: https://oss.sonatype.org/service/local/
/usr/bin/docker run --name c[2](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:2)9c4ce[3](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:3)6771dbeb6[4](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:4)c00ac12f01c7644b269_b97abd --label [5](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:5)c29c4 --workdir /github/workspace --rm -e "INPUT_USERNAME" -e "INPUT_PASSWORD" -e "INPUT_STAGING-PROFILE-ID" -e "INPUT_DESCRIPTION" -e "INPUT_BASE_URL" -e "INPUT_STAGING_PROFILE_ID" -e "HOME" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_REPOSITORY_OWNER_ID" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_REPOSITORY_ID" -e "GITHUB_ACTOR_ID" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKFLOW_REF" -e "GITHUB_WORKFLOW_SHA" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "GITHUB_STATE" -e "GITHUB_OUTPUT" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_ENVIRONMENT" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e "ACTIONS_RESULTS_URL" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/Decompose/Decompose":"/github/workspace" 5c29c4:ce3[6](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:6)[7](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:7)71dbeb64c00ac12f01c7644b26[9](https://github.com/arkivanov/Decompose/actions/runs/6908189767/job/18796938312#step:3:10)
parse error: Invalid numeric literal at line 2, column 0
Error while creating the staging repository.
```

**Comment by arkivanov** [comment]
Version `v1.2.0` seems to work fine.

**Comment by martinbonnin** [comment]
Thanks for sending this and apologies about the inconvenience... What happened is that the `-` was replaced by `_` [a long time ago](https://github.com/nexus-actions/create-nexus-staging-repo/commit/e56509638a34796cb01b95a0c79e280564f305c4).

...but what has changed today is that we've [added type declarations](https://github.com/nexus-actions/create-nexus-staging-repo/pull/8) for [github-workflows-kt](https://github.com/typesafegithub/github-workflows-kt) to generate typesafe Kotlin DSL from the action. And github-workflows-kt requires a tag or branch pointing to the latest major version while we initially had `v1` point to the initial release. So we deleted the `v1` tag and now it's a branch pointing to latest version. I'm not thrilled about it but this seems to be more or less the [GitHub recommendation](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management). You can read more about it [here](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management).

Long story short, if you replace `staging-profile-id` with `staging_profile_id` it should work fine with both  `v1` and `v1.3.0`. Hope this helps

_PS: I'm terrified by everyone using mutable tags/branches for their actions so in case you want to use sha1s instead, shameless plug for a small tool I wrote some time ago that will replace your mutable git refs by sha1s: https://github.com/martinbonnin/pin-github-actions_

**Comment by arkivanov** [comment]
Oh, thanks for the info! I thought the warning is unrelated to the actual issue, so I downgraded to v1.2.0 and fixed the warning at the same time. Will bump to v1.3.0. Thanks again for the great work!
